docker/manylinux-builder/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+docker build . \
+    -f Dockerfile \
+    --build-arg base_image="quay.io/pypa/manylinux2014_x86_64" \
+    -t keyvidev/manylinux-builder && \
+    -t keyvidev/manylinux-builder-x86_64
```

Comment:
I think we don't need to use aliases, as we can simply change GH actions to use new images, so no need for `-t keyvidev/manylinux-builder`

Reply:
Done.